lib/insights-rbac-api-client/version.rb

```diff
@@ -6,10 +6,10 @@
 The version of the OpenAPI document: 1.0.0
 
 Generated by: https://openapi-generator.tech
-OpenAPI Generator version: 4.2.2
+OpenAPI Generator version: 7.1.0-SNAPSHOT
 
 =end
 
 module RBACApiClient
-  VERSION = '1.0.1'
+  VERSION = '1.0.0'
```

Comment:
I think this won't work 😞

Reply:
Thanks! https://github.com/RedHatInsights/insights-rbac-api-client-ruby/pull/16